File: code/py/src/luogu/contest/ioi20230915/qD.py
def main() -> None:
    n, m = map(int, input().split())
    c_score = [None] + list(map(int, input().split()))
    res = 0
    for i in list(map(int, input().split())):
        if i <= n and c_score[i] < 200:
            res += 1
    print(res)


if __name__ == '__main__':
    main()

File: code/java/src/luogu/contest/c20230730njit/QD.java
package luogu.contest.c20230730;

import java.util.*;

class QD {
    static int n;
    static PriorityQueue<int[]> s;
    static void input() {
        Scanner scanner = new Scanner(System.in);
        n = scanner.nextInt();
        s = new PriorityQueue<>(n, Comparator.comparingInt(x -> -x[0]));
        for (int i = 0; i < n; i++) {
            s.offer(new int[]{scanner.nextInt(), i});
        }
    }

    public static void main(String[] args) {
        input();
        int[] vertex;
        int[] res = new int[n];
        for (int i = 0; i < n; i++) {
            vertex = s.poll();

        }
        System.out.println(Arrays.toString(res));
    }
}